File: Log de Programação/2°Bi/Lista 2/SalaC/Lista2-sala_Exercício 2 - Média2.c
#include<stdio.h>

int main()
{
	float n1, n2, M, e, nm;
	
	printf("Digite sua 1 nota: ");
	scanf("%f", &n1);
	printf("Digite sua 2 nota: ");
	scanf("%f", &n2);
	M = (n1 + n2)/2;
	if (M >= 6.0){
		printf("Voce foi aprovado, sua nota foi %.2f", M);
	}
	else {
		printf("Digite sua note no exame: ");
		scanf("%f", &e);
		nm = (M + e)/2;
		if (nm >= 5.0){
			printf("Voce foi aprovado em exame, sua nota foi %.2f", nm);
		}
		else{
			printf("Voce nao foi aprovado, sua nota apos o exame foi %.2f", nm);
		}
	}
}

File: 1 ano/Log de Programação/2°Bi/Lista 2/SalaPy/Lista2-sala_Exercício 2 - Média2.py
n1 = float(input("Digite sua 1° nota: "))
n2 = float(input("Digite sua 2° nota: "))
M = (n1 + n2)/2
if (M >= 6.0):
    print("Você foi aprovado, sua nota foi {:.2f}".format(M))
else:
    e = float(input("Digite sua nota no exame: "))
    nm = (M + e)/2
    if (nm >= 5.0):
        print("Você foi aprovado em exame, sua nota foi {:.2f}".format(nm))
    else:
        print("Você não foi aprovado, sua nota após o exame foi {:.2f}".format(nm))
